app/views/profile/show.html.haml

```diff
@@ -0,0 +1,9 @@
+.inset
+  %h2= "Welcome, #{current_user.full_name} <#{current_user.email_address}>"
+  = form_tag(me_path(current_user), method: :post) do
+    %label.string.control-label First Name
+    %input.form-control{type: 'text', name: 'first_name', value: current_user.first_name}
```

Comment:
I think renaming to `profile` is a nice middle ground! Agree that the default path feels impersonal. I think as a user `me` makes sense but I can imagine a future where we have a bunch of named routes that don't match the controller name and that could get confusing so `profile` would be :tada: